gradle.properties

```diff
@@ -1,5 +1,5 @@
 group = com.cdancy
-version = 0.0.10
+version = 0.0.10-pjr
```

Comment:
Could we set the version to `0.0.10-SNAPSHOT` to enforce the fact that it is not yet released ?